dist/windows/config.nsh

```diff
@@ -83,6 +79,12 @@ XPStyle on
 ; The name of the installer
 Name "qBittorrent ${QBT_INSTALLER_WINDOWNAME}"
 
+; Installer icon
+Icon "qbittorrent.ico"
```

Comment:
I don't object this change. But won't the user confuse it with qbt main executable? 
foobar2000 has a customized icon that shows its small app icon on the lower right bottom and on top of NSIS default icon. I think it is a neat design.